### ISSUE TITLE
"Could not save that post" with code 422

### ISSUE THREAD

**Issue by AamuLumi** [gdpelican/babble#120]
Hi !

I don't know why, but I get a 422 when I try to add a message to chat.

```
POST http://xxx/babble/topics/117/post 422 (Unprocessable Entity)
```

I get the error message "Could not save that port. Try again later" in Babble. However, the post is created in the topic, and a page refreshing update correctly Babble.

Do you have an idea ?

**Comment by MiguelAngelLV** [comment]
Same here.

**Comment by gdpelican** [comment]
Fixed with #121
